PyPowerFlex/base_client.py

```diff
@@ -370,3 +370,17 @@ def _query_selected_statistics(self, action, params=None):
             LOG.error(exc.message)
             raise exc
         return response
+
+    def _modify_entity(self, action, sdc_id, params):
```

Comment:
Lets add some documentation for methods.
Same comment applicable for all files.